tests/utils.go

```diff
@@ -4281,6 +4287,20 @@ func HasCDI() bool {
 	return HasDataVolumeCRD()
 }
 
+func isARM64() bool {
+	virtCli, err := kubecli.GetKubevirtClient()
+	util2.PanicOnError(err)
+	nodes := util2.GetAllSchedulableNodes(virtCli).Items
+	Expect(nodes).ToNot(BeEmpty(), "There should be some node")
+	return nodes[0].Status.NodeInfo.Architecture == "arm64"
```

Comment:
I think we can assume that the architecture of the nodes does not change during the suite exectuion. Maybe it makes sense to only fetch this once upfront. Maybe in `BeforeTestSuite`.

Reply:
OK

Reply:
I put the get CPU arch in `SynchronizedBeforeTestSetup`, thus it would run only once.